tron/serialize/runstate/statemanager.py

```diff
@@ -133,13 +133,13 @@ def restore(self, job_names):
         }
         return state
 
-    def _restore_runs_for_job(self, job_name, job_state):
+    def _restore_runs_for_job(self, job_name, job_state, read_json: bool = False):
         """Restore the state for the runs of each job"""
         run_nums = job_state["run_nums"]
         keys = [jobrun.get_job_run_id(job_name, run_num) for run_num in run_nums]
-        job_runs_restored_states = self._restore_dicts(runstate.JOB_RUN_STATE, keys)
-        runs = copy.copy(job_runs_restored_states)
-        for run_id, state in runs.items():
+        job_runs_restored_states = self._restore_dicts(runstate.JOB_RUN_STATE, keys, read_json)
+        restored_states_copy = copy.copy(job_runs_restored_states)
+        for run_id, state in restored_states_copy.items():
```

Comment:
ignorable suggestion: 
```suggestion
        all_job_runs = copy.copy(job_runs_restored_states)
        for run_id, state in all_job_runs.items():
```
the suggested naming would make it clearer what this holds :)